mutiny-core/src/nodemanager.rs

```diff
@@ -306,10 +307,50 @@ impl From<bdk::TransactionDetails> for TransactionDetails {
     }
 }
 
+/// Information about a channel that was closed.
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, Hash)]
+pub struct ChannelClosure {
+    pub user_channel_id: Option<[u8; 16]>,
+    pub channel_id: Option<[u8; 32]>,
+    pub node_id: Option<PublicKey>,
+    pub reason: String,
+    pub timestamp: u64,
+}
+
+impl ChannelClosure {
+    pub fn new(
+        user_channel_id: u128,
+        channel_id: [u8; 32],
+        node_id: Option<PublicKey>,
+        reason: ClosureReason,
+    ) -> Self {
+        Self {
+            user_channel_id: Some(user_channel_id.to_be_bytes()),
+            channel_id: Some(channel_id),
+            node_id,
+            reason: reason.to_string(),
+            timestamp: utils::now().as_secs(),
+        }
+    }
+}
+
+impl PartialOrd for ChannelClosure {
+    fn partial_cmp(&self, other: &Self) -> Option<core::cmp::Ordering> {
+        Some(self.cmp(other))
+    }
+}
+
+impl Ord for ChannelClosure {
+    fn cmp(&self, other: &Self) -> core::cmp::Ordering {
+        self.timestamp.cmp(&other.timestamp)
+    }
+}
```

Comment:
Should update the test for activity list sorting.

Reply:
Added